components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -294,8 +294,24 @@ func CheckAndMountCACertBundle(ctx context.Context, cli client.Client, notebook
 	workbenchConfigMap := &corev1.ConfigMap{}
 	err := cli.Get(ctx, client.ObjectKey{Namespace: notebook.Namespace, Name: workbenchConfigMapName}, workbenchConfigMap)
 	if err != nil {
-		log.Info("workbench-trusted-ca-bundle ConfigMap is not present, skipping mounting of certificates.")
-		return nil
+		log.Info("workbench-trusted-ca-bundle ConfigMap is not present,start creating..")
+		// create the ConfigMap if it does not exist
```

Comment:
We don't want to utilize the `notebook_controller.CreateNotebookCertConfigMap` function for this?

Reply:
That func createdNotebookCertConfigMap has pointer to reconciler.
as we want to handle the admission, we didn't use that func.

please do share if you feel it can be used otherwise.